generic/piranha/src/models/outgoing_edges.rs

```diff
@@ -16,7 +16,7 @@ use serde_derive::Deserialize;
 #[derive(Deserialize, Debug, Clone, Hash, PartialEq, Eq, Default)]
 // Represents the `edges.toml` file
 pub(crate) struct Edges {
-  pub edges: Vec<OutgoingEdges>,
+  pub(crate) edges: Vec<OutgoingEdges>,
```

Comment:
are these visibility changes made because of errors?

Reply:
`pub` makes it visible everywhere including the external world.
`pub (crate)` makes it visible only within the crate (not to the external)